stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/SQLDatasourceUtils.java

```diff
@@ -94,6 +97,7 @@
     public static final String POSTGRES_OID_COLUMN_TYPE_NAME = "oid";
     private static final int ORACLE_CURSOR_TYPE = -10;
     private static final String TIME_FIELD = "time";
+    private static final String POOL_MAP_KEY = UUID.randomUUID().toString();
```

Comment:
Shall we move this to Constants.java class. (Similar to https://github.com/Manuri/ballerina/blob/connection-pool-change/stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/Constants.java#L199)

Reply:
This is not used anywhere else and we don't need to make this key public so that anyone who is doing a pool related code change will always have to go through the two methods provided in this class to add/retrieve pool.